wolfssl/wolfcrypt/settings.h

```diff
@@ -1103,6 +1110,9 @@ extern void uITRON4_free(void *p) ;
     defined(WOLFSSL_STM32L4)
 
     #define SIZEOF_LONG_LONG 8
+    #ifndef CHAR_BIT
+      #define CHAR_BIT SIZEOF_LONG_LONG
```

Comment:
CHAR_BIT is the size of a primitive. I think it is OK for it to be set to 8 here and not be dependent on the size in bytes of a larger type.